scripts/install.sh

```diff
@@ -0,0 +1,179 @@
+#!/bin/bash
+# Bridge to kubernetes installer
+#
+#  ____       _     _              _          _  __     _                          _
+# | __ ) _ __(_) __| | __ _  ___  | |_ ___   | |/ /   _| |__   ___ _ __ _ __   ___| |_ ___  ___
+# |  _ \| '__| |/ _` |/ _` |/ _ \ | __/ _ \  | ' / | | | '_ \ / _ \ '__| '_ \ / _ \ __/ _ \/ __|
+# | |_) | |  | | (_| | (_| |  __/ | || (_) | | . \ |_| | |_) |  __/ |  | | | |  __/ ||  __/\__ \
+# |____/|_|  |_|\__,_|\__, |\___|  \__\___/  |_|\_\__,_|_.__/ \___|_|  |_| |_|\___|\__\___||___/
+#                    |___/
+# usage:
+#    curl -fsSL https://raw.githubusercontent.com/Azure/Bridge-To-Kubernetes/main/scripts/install.sh | bash
+set -e
```

Comment:
WSL2 is part of windows but it is still a tool to get linux on windows. I believe this feature of using bridge through CLI should work for windows as well.

Reply:
In other words, why are we choosing to add this constraint to this feature for windows users? The constrain being we will be forcing the windows user to install/set up WSL2 on their machine just so they can download/install bridge. Keep in mind setting up WSL/WSL2 is not a small thing right? It does a lot of changes on the machine and in fact lot of people will disable a lot of this hyper-v features.

Reply:
for instance why not use powershell script instead? powershell will work in all OS.

Reply:
@elenavillamil Thank you for your thoughts and comments. To use power shell in Mac OS, users have to install them manually, here are the instructions - [power shell installation on MAC OS](https://learn.microsoft.com/en-us/powershell/scripting/install/installing-powershell-on-macos?view=powershell-7.2).  The same can be done for bash for windows, you can install Git Bash in windows if users don't want to install WSL2 , here are the instructions - [Git bash for windows](https://gitforwindows.org/). I am happy to discuss more on this :)

Regards,
Hari

Reply:
> > We need to add a check on dsc (bridge code) on what version of itself is running. So when people run dsc connect it will show a warning if version is old and it will exit and say run "..." to upgrade if it is too old and we want to force user to upgrade before bridge works.
> 
> Yes, this can be done, I feel it is separate feature/PR. Happy to discuss more.

It is part of same feature as this is needed to make this feature complete. This PR or different PR we can discuss further. I do prefer it as part of this PR but open to it being on a different one.